### ISSUE TITLE
Improve error handling when github account won't connect correctly

### ISSUE THREAD

**Issue by SusanBrown** [cwrc/CWRC-GitWriter#53]
NB there may be an older issue related to this but I couldn't find it. 

## Expected Behaviour
I don't fully understand why this happens to me all the time, but when CWRC-Writer can't connect to my github account correctly I would like better feedback and an avenue for dealing with the problem that doesn't require me to go into my browser settings and erase all of my cookies.

## Current Behaviour
What I currently see is this.
![image.png](https://images.zenhubusercontent.com/5a60ec3a4b5806bc2bc34e2e/a9fc778c-2e34-4e5e-92c6-4a6eceeb31eb)
and then this when I try to find a document I could possibly work with:
![image.png](https://images.zenhubusercontent.com/5a60ec3a4b5806bc2bc34e2e/5d2ea572-5e69-4cde-8f2f-da85da3cbf31)
In other words, gitwriter falsely implies that I should be able to do stuff from this screen that I cannot do.

## Possible Solution
Since I can't actually do anything on any of the tabs for this screen, I wonder if we could create a variant upon the following screen:
![image.png](https://images.zenhubusercontent.com/5a60ec3a4b5806bc2bc34e2e/f842c684-661d-44e4-ab90-1ce987df3447)

**Comment by SusanBrown** [comment]
I'm not sure exactly how the problem should be articulated to the user, but I think we need both clearer error feedback and, ideally, some way to deal with the problem. Open to suggestions if I've misunderstood what's going on, but what about something like this:

                                                                Problem authenticating with GitHub

CWRC-Writer has run into a problem that may have to do with your GitHub authentication. We suggest logging out of GitHub and logging back in again to see if this will allow CWRC-Writer to connect properly to your account.

If that doesn't work, we suggest you clear your cookies in your browsing data (cookies) and try again.

Log out of GitHub [green button]

then, after a user clicks on that button, and Git-Writer logs them out of github, CWRC-Writer could do what it usually does when someone clicks on "Authenticate with GitHub"--takes it to the login screen that then takes them to CWRC-Writer after they log in.

I realize that it may not be allowable for a tool to log someone out of github. If so, can we provide much more explicit instructions along the lines of 

                                      Problem authenticating with GitHub

CWRC-Writer has run into a problem that may have to do with your GitHub authentication. We suggest logging out of GitHub and logging back in again to see if this will allow CWRC-Writer to connect properly to your account. Once you have done this, click on the button below to try again. If that doesn't work, we suggest you clear your cookies in your browsing data (cookies) and try again.

Reauthenticate with GitHub [green button]

**Comment by ajmacdonald** [comment]
Ideally this shouldn't be happening at all. I'd like to start by trying to discover why you are experiencing this issue @SusanBrown

**Comment by SusanBrown** [comment]
It's very hard to say. I think we tried to figure out out with James but couldn't so that's why I was approaching it from a "dealing with the symptoms" perspective, but it's worth another go at trying to figure it out, if you think it's possible we could. I can't reproduce it, though. It recurs regularly for me. I can be logged into and using GitHub regularly, but somehow CWRC-Writer loses the ability to access what it needs to know from my browser (Chrome) in order to authenticate me. All the tabs are blank and even searching for existing documents in other repos brings up nothing. What fixed it for me was clearing all my cookies. Logging out of and then back into Github didn't do fix it.

**Comment by ilovan** [comment]
@SusanBrown , we added a "log out" button to the Git-Writer toolbar. If I understand how that works, it clears the GitWriter authentication cookies, so it may work to fix this problem next time when you encounter this problem (@ajmacdonald , please jump in if I am wrong).
If what I am suggesting works, we should create an error message that describes this step.
